.github/workflows/Release.yml

```diff
@@ -27,16 +27,16 @@ jobs:
           node-version: 16
 
       - name: Install Dependencies
-        run: yarn
+        run: pnpm install
```

Comment:
I think `pnpm/action-setup` is necessary.

Reply:
I couldn't do this perfectly😇
fixed!
https://github.com/sveltejs/svelte-eslint-parser/pull/334/commits/92a644bd23fced4017c9baea3bfd12a72d3587f1